assets/src/edit-story/components/canvas/selection.js

```diff
@@ -51,6 +51,12 @@ function Selection() {
     return null;
   }
 
+  // No need for displaying non-functional frame for selected background.
+  const isVideoPlaceholder = selectedElements[0]?.resource?.isPlaceholder;
```

Comment:
The comment probably needs updating :)

Is this possible to happen for multi-selection as well, or is it always single-selection in case of a placeholder? Currently, it's checking for just the first selected element.

Reply:
(Ok, seeing now that it can only be in the single selection)

Reply:
Should the video placeholder be removed from the `otherNodes` as well from the elements that can be snapped to for `useSnapping` ?

Reply:
Hmm good question. Since the dimensions are the same as the canvas, perhaps it doesn't really make a difference?